index.html

```diff
@@ -35,9 +35,9 @@ <h2 id="planet-name"></h2>
         <a href="" id="planet-info-source"></a>
 
         <div class="planet-portrait__board__buttons">
-          <button onclick="selectInfoGroup('overviewContent')">OVERVIEW</button>
-          <button onclick="selectInfoGroup('internalStructure')">INTERNAL STRUCTURE</button>
-          <button onclick="selectInfoGroup('surfaceGeology')">SURFACE GEOLOGY</button>
+          <button id="btn-overviewContent" onclick="selectInfoGroup('overviewContent')">OVERVIEW</button>
+          <button id="btn-internalStructure" onclick="selectInfoGroup('internalStructure')">INTERNAL STRUCTURE</button>
+          <button id="btn-surfaceGeology" onclick="selectInfoGroup('surfaceGeology')">SURFACE GEOLOGY</button>
```

Comment:
id's of the sections buttons
<img width="432" alt="image" src="https://user-images.githubusercontent.com/26731448/180101016-eac214cc-bccf-4886-b202-53b52da1af2d.png">